app/assets/javascripts/discourse-common/lib/deprecated.js.es6

```diff
@@ -1,3 +1,11 @@
-export default function deprecated(msg) {
-  console.warn(`DEPRECATION: ${msg}`); // eslint-disable-line no-console
+export default function deprecated(msg, opts = {}) {
+  msg = ['Deprecation notice:', msg];
+  if (opts.since) { msg.push(`(deprecated since Discourse ${opts.since})`; }
+  if (opts.dropFrom) { msg.push(`(removal in Discourse ${opts.dropFrom})`; }
```

Comment:
Missing a close bracket here, and on the line above.

Reply:
Fixed